### DISCUSSION TITLE
code 1000

### DISCUSSION THREAD

**Discussion by VladKost999** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#62] [Q&A]
How i can solve code 10000?
![image](https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/assets/91079045/c33b7c71-f323-45a1-b6f3-5b0654b3e19f)

**Comment by 1379I** [comment]
![image](https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/assets/85945950/baced9b8-8286-45db-8fea-184c29e2e9ef)
 same error

**Comment by murdegern** [comment]
You need to downgrade your settings app. See here:

https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/issues/82
